### PR TITLE
update type definitions up to Bootstrap 5.0.2

### PR DESCRIPTION
**Pull request by techfox7**
I updated  type definitions up to Bootstrap 5.0.2

### CODE REVIEW

File: types/bootstrap/js/dist/alert.d.ts [hunk 1]
```diff
@@ -3,17 +3,31 @@ import BaseComponent from "./base-component";
...
+     * Closes an alert by removing it from the DOM. If the .fade and .show
+     * classes are present on the element, the alert will fade out before it
+     * is removed.
+     */
+     close(): void;
+
+     /**
+      * Destroys an element's alert. (Removes stored data on the DOM element)
+      */
+     dispose(): void;
```
Comment (martin-badin) [inline types/bootstrap/js/dist/alert.d.ts:17]: The `BaseComponent` includes `dispose` function. Please remove it from all components.

**Review by martin-badin — CHANGES_REQUESTED** [review]

File: types/bootstrap/js/dist/alert.d.ts (hunk 1 above)
Comment (techfox7) [reply types/bootstrap/js/dist/alert.d.ts]: Sorry, I just removed it from all components. Thanks.

Comment (techfox7) [reply types/bootstrap/js/dist/alert.d.ts]: and already changed and committed, pushed it.

File: types/bootstrap/index.d.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for bootstrap 5.0
+// Type definitions for bootstrap 5.0.2
```
Comment (peterblazejewicz) [inline types/bootstrap/index.d.ts:1]:
```suggestion
// Type definitions for bootstrap 5.0
```
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library

File: types/bootstrap/js/dist/carousel.d.ts [hunk 3]
```diff
@@ -39,12 +39,19 @@ declare class Carousel extends BaseComponent {
      */
     to(index: number): void;
 
-    /**
+     /**
```
Comment (peterblazejewicz) [inline types/bootstrap/js/dist/carousel.d.ts:42]:
```suggestion
    /**
```

File: types/bootstrap/js/dist/dropdown.d.ts [hunk 4]
```diff
@@ -24,13 +24,20 @@ declare class Dropdown extends BaseComponent {
      */
     update(): void;
 
-    /**
+     /**
```
Comment (peterblazejewicz) [inline types/bootstrap/js/dist/dropdown.d.ts:27]:
```suggestion
    /**
```

**Comment by peterblazejewicz** [comment]
@techfox7 can you please run `npx prettier --write types/bootstrap` to unify formatting? That's OK for me to review with formatting changes incuded. thx!

**Comment by techfox7** [comment]
> @techfox7 can you please run `npx prettier --write types/bootstrap` to unify formatting? That's OK for me to review with formatting changes incuded. thx!

I did run "npx prettier --write types/bootstrap' to unify formatting. thx!

**Comment by techfox7** [comment]
Fixed all.   I check 'npm run test' is working correctly.

File: types/bootstrap/v4/bootstrap-tests.ts [hunk 5]
```diff
@@ -5,36 +5,36 @@ declare let aHtmlElement: HTMLElement;
...
 
 // $ExpectType JQuery<HTMLElement>
-$("#carousel").carousel("pause");
+$('#carousel').carousel('pause');
 
-$("#carousel").carousel(100);
+$('#carousel').carousel(100);
 
-$("#carousel").on("slide.bs.carousel", function(e) {
+$('#carousel').on('slide.bs.carousel', function (e) {
```
Comment (peterblazejewicz) [inline types/bootstrap/v4/bootstrap-tests.ts:37]:
```suggestion
$('#carousel').on('slide.bs.carousel', function slideHandler(e)(e) {
```

**Review by peterblazejewicz — APPROVED** [review]
LGMT!
@techfox7 thx!

**Review by peterblazejewicz — APPROVED** [review]

**Pull request closed without merging**